### PR TITLE
load test for migration

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -44,7 +44,10 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
-	dynamicClient, err := dynamic.NewForConfig(ctx.KubeConfig)
+	dynamicClientConfig := dynamic.ConfigFor(ctx.KubeConfig)
+	dynamicClientConfig.Burst = 40
```
Comment (p0lyn0mial) [inline pkg/operator/starter.go]: we need to experiment and find some good numbers for QPS/Burst

**Comment by p0lyn0mial** [comment]
this pull requires https://github.com/openshift/library-go/pull/595

File: pkg/operator/starter.go (hunk 1 above)
Comment (sttts) [reply pkg/operator/starter.go]: What does kube-storage-version-migrator use?

Comment (sttts) [reply pkg/operator/starter.go]: Please rename `dynamicClient` to `migrationClient` to make clear it is not meant for anything else than migration.

File: test/e2e-encryption/encryption_test.go [hunk 2]
```diff
@@ -13,6 +13,7 @@ import (
 )
 
 func TestEncryptionTypeIdentity(t *testing.T) {
+	t.SkipNow()
```
Comment (sttts) [inline test/e2e-encryption/encryption_test.go]: WIP?

File: pkg/operator/starter.go (hunk 1 above)
Comment (p0lyn0mial) [reply pkg/operator/starter.go]: > What does kube-storage-version-migrator use?

it seems like they use default settings https://github.com/kubernetes-sigs/kube-storage-version-migrator/blob/master/cmd/migrator/app/migrator.go#L58

**Pull request merged**